common/api/core-backend.api.md

```diff
@@ -1838,11 +1839,13 @@ export class ECSqlStatement implements IterableIterator<any>, IDisposable {
     bindValues(values: any[] | object): void;
     clearBindings(): void;
     dispose(): void;
+    // @internal (undocumented)
+    formatCurrentRow(currentResp: any, rowFormat?: QueryRowFormat): any[] | object;
     getBinder(parameter: string | number): ECSqlBinder;
     getColumnCount(): number;
     // @internal
     getNativeSql(): string;
-    getRow(): any;
+    getRow(args?: IModelJsNative.ECSqlRowArg): any;
```

Comment:
You cannot have a non-`@internal` API that uses anything from `IModelJsNative`. `IModelJsNative` is inherently `@internal` and unstable.

Reply:
To address this, I am considering declaring an `ECSqlRowArg` interface with the same properties in `core\backend\src\ECSqlStatement.ts`. However, this approach would result in code duplication between `IModelJsNative` and `itwinjs-core`.
Please advise on the best approach to handle this situation.

Reply:
Alternative is to put `ECSqlRowArg` into core-common. See the other types like `QueryRowFormat` that are implemented by NativeLibrary.ts from core-common.

You would still need to temporarily duplicate the interface in NativeLibrary.ts, until the PRs merge and you can update the native package to use the new version of core-common containing the interface definition.

Reply:
Added `ECSqlRowArg` interface to core-common